assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -133,6 +135,15 @@ class PageEdit extends Component {
 		} );
 	}
 
+	componentDidUpdate( prevProps ) {
+		if (
+			VIDEO_BACKGROUND_TYPE === this.props.attributes.mediaType &&
+			this.props.attributes.mediaUrl !== prevProps.attributes.mediaUrl &&
+			this.videoPlayer.current ) {
```

Comment:
The `)` and `}` could probably be on the next line for readability.

Reply:
Totally. Looks like ESLint doesn't catch these things 🤷‍♂